test/e2e-go/upgrades/application_support_test.go

```diff
@@ -155,8 +156,6 @@ int 1
 	a.NoError(err)
 	signedTxn, err := client.SignTransactionWithWallet(wh, nil, tx)
 	a.NoError(err)
-	round, err = client.CurrentRound()
```

Comment:
Why drop this?

Reply:
not used below until the next client.CurrentRound() invocation